plugins/lsp/language-servers/default.nix

```
@@ -545,6 +545,11 @@ with lib; let
       description = "Enable taplo, for TOML";
       package = pkgs.taplo;
     }
+    {
+      name = "templ";
+      description = "Enable the templ language server for the templ HTML templating language";
+      package = pkgs.templ;
+    }
```

```suggestion
      description = "Enable the templ language server for the templ HTML templating language";
    }
```
As the package has the same name as the LS, you don't need to specify it.